package/firmware/ath11k-firmware/Makefile

```diff
@@ -62,11 +62,25 @@ define Download/qcn9074-board
 endef
 $(eval $(call Download,qcn9074-board))
 
+QCN6018_BOARD_FILE:=board-2.bin
+
+define Download/qcn6018-board
+  URL:=https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/plain/ath11k/IPQ6018/hw1.0/board-2.bin
```

Comment:
Please package this in the `package/firmware/linux-firmware/qca_ath11k.mk` file. It already handles the content of the `linux-firmware` repository.

Reply:
Yeah, that wont work on ath11k, the reason why there is no generic BDF is that you cannot use generic ones on non QCA reference boards.

Reply:
i found that it worked and the WiFi came up but that it would sometimes crash.  I can't report to you the error right now.

Reply:
Again, you gotta add your custom BDF-s for the board, package them as board-2.bin with a proper variant so they are matched to this board.

The public ath11k FW for IPQ6018 is outdated and rather unstable but QCA seems like they dont want to release the current 2.9 FW for some reason.

Reply:
I extracted the firmware from the factory image and found the boardData files but not sure which ones to include in the board.json when using bdencoder.  I am expecting some help in this regard on Monday.  In the meantime I also found a bdwlan.bin in the firmware and copied that onto the board running OpenWRT and it̀s working fantastic.  I guess that file isǹt sufficient though for uptream...?